src/services/codefixes/importFixes.ts

```diff
@@ -623,6 +647,30 @@ function getSingleExportInfoForSymbol(symbol: Symbol, symbolName: string, module
     }
 }
 
+function getImportFixesForNonExportedSymbols(
+    sourceFile: SourceFile,
+    symbol: Symbol,
+    importKind: ImportKind,
+    program: Program,
+    isValidTypeOnlyUseSite: boolean,
+    targetFilePath: string,
+    importMap = createExistingImportMap(program.getTypeChecker(), sourceFile, program.getCompilerOptions()),
+): { computedWithoutCacheCount: number; fixes: readonly ImportFixWithModuleSpecifier[]; } {
+    const checker = program.getTypeChecker();
+    const symbolInfo = [{
+        symbol,
+        targetFlags: SymbolFlags.Value,
+        importKind,
+        targetFilePath,
+    }];
+    const existingImports = flatMap(symbolInfo, importMap.getImportsForNonExportedSymbols);
+    const addToExisting = tryAddToExistingImport(existingImports, isValidTypeOnlyUseSite, checker, program.getCompilerOptions());

```

Comment:
Assert that this is defined so that it is easy to track later on in case something goes wrong.

Reply:
Which one are you pointing at?